source/onboarding/person/index.html.md.erb

```diff
@@ -9,6 +9,8 @@ weight: 2
 
 ## Azure AD groups
 
+Azure AD is used to give you and your team-mates access to specific subscriptions and resources in Azure. When your github user has access to a subscription, you will be able to view, create and modify Azure resources for your team.
```

Comment:
use sentence per line to make diff's easier to read, and use the product's display name, GitHub, not github
```suggestion
Azure AD is used to give you and your team-mates access to specific subscriptions and resources in Azure.
When your GitHub user has access to a subscription, you will be able to view, create and modify Azure resources for your team.
```